sourced/ml/cmd_entries/repos2coocc.py

```diff
@@ -3,24 +3,20 @@
 
 from sourced.ml.extractors import IdentifiersBagExtractor
 from sourced.ml.models import OrderedDocumentFrequencies
-from sourced.ml.transformers import Ignition, HeadFiles, UastExtractor, Cacher, UastDeserializer, \
-    CooccConstructor, CooccModelSaver, BagFeatures2DocFreq, Uast2BagFeatures, Counter, \
-    UastRow2Document
-from sourced.ml.utils import create_engine
+from sourced.ml.transformers import Cacher, UastDeserializer, CooccConstructor, CooccModelSaver, \
+    BagFeatures2DocFreq, Uast2BagFeatures, Counter, UastRow2Document, create_uast_source
 from sourced.ml.utils.engine import pipeline_graph, pause
 
 
 @pause
 def repos2coocc_entry(args):
     log = logging.getLogger("repos2coocc")
-    engine = create_engine("repos2coocc-%s" % uuid4(), **args.__dict__)
     id_extractor = IdentifiersBagExtractor(docfreq_threshold=args.min_docfreq,
                                            split_stem=args.split_stem)
```

Comment:
There is no argument `--split-stem`. https://github.com/src-d/ml/pull/220/files - this is the PR that broke cooc extraction because `split-stem` was replaced with `split` and in `repos2coocc_entry` it's still `split_stem=args.split_stem`

Reply:
okay, changing to split